.github/workflows/continuous-deploy.yml

```diff
@@ -229,6 +272,9 @@ jobs:
       id-token: write # Required to authenticate with AWS
       contents: read # Required to clone this repository
     runs-on: ubuntu-latest
+    concurrency:
+      group: cicd-deploy
+      cancel-in-progress: false
```

Comment:
similar to cicd-build - shouldn't we cancel in progress deploy jobs if new push comes in?

Reply:
Not currently no. We don't want to cancel an actual deployment halfway through the process.